executor/seqtest/seq_executor_test.go

```diff
@@ -802,10 +925,16 @@ func (c *checkPrioClient) getCheckPriority() pb.CommandPri {
 
 func (c *checkPrioClient) SendRequest(ctx context.Context, addr string, req *tikvrpc.Request, timeout time.Duration) (*tikvrpc.Response, error) {
 	resp, err := c.Client.SendRequest(ctx, addr, req, timeout)
-	switch req.Type {
-	case tikvrpc.CmdCop:
-		if c.getCheckPriority() != req.Priority {
-			return nil, errors.New("fail to set priority")
+	c.mu.RLock()
+	defer func() {
+		c.mu.RUnlock()
+	}()
+	if c.mu.checkPrio {
```

Comment:
Why do we need this flag?

Reply:
Some SQLs in bootstrap and teardown will be set to high priority. But the `getCheckPriority` will always return a normal priority.